homeassistant/components/microsoft_face/__init__.py

```diff
@@ -98,7 +98,9 @@ async def async_create_group(service):
             entities[g_id] = MicrosoftFaceGroupEntity(hass, face, g_id, name)
             entities[g_id].async_write_ha_state()
         except HomeAssistantError as err:
-            _LOGGER.error("Can't create group '%s' with error: %s", g_id, err)
+            msg = f"Can't create group '{g_id}' with error: {err}"
+            _LOGGER.error(msg)
+            raise HomeAssistantError(msg)
```

Comment:
```suggestion
            raise
```

Reply:
Ok. Got it.
I updated the code. Thanks for the advise.